src/components/Onboarding/WalletKey/index.js

```diff
@@ -22,13 +24,12 @@ class TermsOfUse extends Component {
   handleExportingPrivateKey = () => {
     Auth.currentSession({ bypassCache: true })
       .then(data => {
-        API.get("Get Service", "/signup", {
-          headers: {
-            Authorization: data.idToken.jwtToken,
-          },
-        })
+        const apiName = APIEndpoints.USER.name;
+        const apiPath = APIPaths.SIGNUP;
+        const myInit = generateAPIInit(data.idToken.jwtToken);
```

Comment:
`generateAPIInit` doesn't convey what exactly it is returning. Could you try to rename it to make it more explicit. I couldn't guess it.

Reply:
renamed in [commit](https://github.com/singnet/refactored-dapp/pull/116/commits/e2534835fc338320754a1cf266653f76d6c5b3b1)

Reply:
We could also rename the `myInit` variable in all the places.

Reply:
[renamed](https://github.com/singnet/refactored-dapp/pull/116/commits/6b733d8a9fb5159092165074ce784b09bb4fba89)